lib/omniauth/strategies/facebook.rb

```diff
@@ -9,18 +9,15 @@ module Strategies
     class Facebook < OmniAuth::Strategies::OAuth2
       class NoAuthorizationCodeError < StandardError; end
 
-      DEFAULT_SCOPE = 'email'
+      DEFAULT_SCOPE = 'email'.freeze
+      API_VERSION = 'v2.6'.freeze
 
       option :client_options, {
-        :site => 'https://graph.facebook.com',
+        :site => "https://graph.facebook.com/#{API_VERSION}",
         :authorize_url => "https://www.facebook.com/dialog/oauth",
```

Comment:
I'm looking here:

> Versioned paths aren't just true for API endpoints, they're also true for dialogs and social plugins. For example, if you want to generate the Facebook Login dialog for a web app, you can prepend a version number to the endpoint that generates the dialog.
>
> https://developers.facebook.com/docs/apps/versions (under Dialogs)

and in this Gem's README:

https://github.com/mkdynamic/omniauth-facebook#api-version (https://github.com/mkdynamic/omniauth-facebook/commit/9f729037e5c27d102462ebbb205b5da17f9aaf86)

I have not verified it though.